### ISSUE TITLE
Advanced Command Module, a Gemini alternative, has considerably less tank volume

### ISSUE THREAD

**Issue by jwvanderbeck** [KSP-RO/RealismOverhaul#2275]
Given that the Advanced Command Pod (`ROAdvCapsule`) is presented as an alternative part option for a Gemini style 2 crew program, uses the same training in RP1, costs the same as Gemini in ROCapsules (`ROC-GeminiCM`), and shares the unlock cost, and both more massive and physically larger, it seems really odd that it has nearly **1/10th** the internal SM volume.  Especially when you consider that the Gemini part also has an external SM in addition to the CM yet still has 525l of internal SM volume compared to the Advanced Command Pod's measly 60l

tl;dr why does the Advanced Command Pod have only 60l internal SM volume while the Gemini from ROCapsules has 525l?
